wp-api.js

```diff
@@ -434,34 +380,39 @@
 	/**
 	 * Backbone model for revisions
 	 */
-	wp.api.models.Revision = wp.api.models.Post.extend(
+	wp.api.models.Revision = BaseModel.extend( _.extend(
```

Comment:
I'm not a JSDoc expert, but should we somehow mention that these models get the mixins in the doc block?